x/fee_grant/internal/keeper/keeper.go

```diff
@@ -0,0 +1,121 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/fee_grant/exported"
+	"github.com/cosmos/cosmos-sdk/x/fee_grant/internal/types"
+)
+
+type Keeper struct {
```

Comment:
nit, missing godoc

Reply:
Also, we should add a `Logger` function as the other modules do as well

Reply:
ok